### ISSUE TITLE
using cached_video_player in pageview.builder

### ISSUE THREAD

**Issue by GeeseBumps** [vikram25897/flutter_cached_video_player#32]
First of all, appreciate for your effort!

I am wondering is it working right in my app.
I am using pageview.builder and there are several short videos in each page.
However, I cannot feel the difference between using original video_player and this package.

I thought, video which I watched should be cached, so even if swiped to another page the videos that i watched should be loaded instantly because they are cached. However, videos I watched are loading as same when I used normal video_player.

Am I thinking wrong? How can I test if videos are really cached?

**Comment by vinothvino42** [comment]
> First of all, appreciate for your effort!
> 
> I am wondering is it working right in my app.
> I am using pageview.builder and there are several short videos in each page.
> However, I cannot feel the difference between using original video_player and this package.
> 
> I thought, video which I watched should be cached, so even if swiped to another page the videos that i watched should be loaded instantly because they are cached. However, videos I watched are loading as same when I used normal video_player.
> 
> Am I thinking wrong? How can I test if videos are really cached?

I think you're making UI like the TikTok kind of app. I'm using cached video player and preload page view package to work smoothly

**Comment by rakesh4577** [comment]
@GeeseBumps  you are right I have tested the beta version on Oneplus Nord it's actually not caching the video,  the video is getting download again and again, can someone help us out? @vikram25897

**Comment by vikram25897** [comment]
@GeeseBumps can you post a testable example?

**Comment by rakesh4577** [comment]
@vinothvino42  hey are you using `2.0.0-dev.1` version or non null safety ? because i am using `2.0.0-dev.1` version with preloadpageview and its not caching video?

**Comment by vinothvino42** [comment]
> preloadpageview

I'm using `preload_page_view: ^0.1.6` and `preloadPagesCount = 2`

**Comment by rakesh4577** [comment]
@vinothvino42  and which version of `flutter_cached_video_player`?
